docs/en/pact-functions.md

```diff
@@ -2104,14 +2104,6 @@ pact> (expect-that "addition" (> 2) (+ 1 2))
 ```
 
 
-### format-address {#format-address}
```

Comment:
What does dropping this have to do with WebAuthn signatures?

Reply:
Dropping it was required after removing a method from the `Scheme` typeclass that `format-address` required.

We have to remove some methods and associated types from `Scheme` to accommodate webauthn. It's believed be to be unused or only used very rarely.